drivers/sensor/st/lsm6dsv16x/lsm6dsv16x_trigger.c

```diff
@@ -246,6 +264,47 @@ static void lsm6dsv16x_work_cb(struct k_work *work)
 }
 #endif /* CONFIG_LSM6DSV16X_TRIGGER_GLOBAL_THREAD */
 
+#if DT_ANY_INST_ON_BUS_STATUS_OKAY(i3c)
+static int lsm6dsv16x_ibi_cb(struct i3c_device_desc *target,
+			  struct i3c_ibi_payload *payload)
+{
+	const struct device *dev = target->dev;
+	struct lsm6dsv16x_data *lsm6dsv16x = dev->data;
+
+	/*
+	 * The IBI Payload consist of the following 10 bytes :
+	 * 1st byte: MDB
+	 * - MDB[0]: FIFO interrupts (FIFO_WTM_IA, FIFO_OVR_IA, FIFO_FULL_IA, CONTER_BDR_IA)
+	 * - MDB[1]: Physical interrupts (XLDS, GDA, TDA, XLDA_OIS, GDA_OIS)
+	 * - MDB[2]: Basic interrupts (SLEEP_CHANGE_IA, D6D_IA, DOUBLE_TAP, SINGLE_TAP, WU_IA,
+	 *           FF_IA)
+	 * - MDB[3]: SHUB DRDY (SENS_HUB_ENDOP)
+	 * - MDB[4]: Advanced Function interrupt group
+	 * - MDB[7:5]: 3'b000: Vendor Definied
+	 *             3'b100: Timing Information
+	 * 2nd byte: FIFO_STATUS1
+	 * 3rd byte: FIFO_STATUS2
+	 * 4th byte: ALL_INT_SRC
+	 * 5th byte: STATUS_REG
+	 * 6th byte: STATUS_REG_OIS
+	 * 7th byte: STATUS_MASTER_MAIN
+	 * 8th byte: EMB_FUNC_STATUS
+	 * 9th byte: FSM_STATUS
+	 * 10th byte: MLC_STATUS
+	 */
```

Comment:
Sorry, I'm not able to find this anywhere on the public documentation (AN and datasheet). I could ask here internally to the design team, but I think that in zephyr we should make visible only whatever is public. Am I missing something? Can you point me to the proper doc?

Reply:
I checked with design team. It seems that information is currently only internal, maybe shared with some specific customer. Nevertheless, it seems that we may publish it in the future on the AN (not sure why it is not there already...). I'm still investigating about it...